test/activation_based/test_op_counter_neuromc_runtime.py

```
@@ -1121,3 +1121,61 @@ def test_neuromc_exact_multi_step_batchnorm2d_uses_true_channel_dim():
+    class FunctionalConv(nn.Module):
+        def __init__(self):
+            super().__init__()
+            self.weight = nn.Parameter(torch.randn(3, 2, 3, 3))
+
+        def forward(self, x):
+            return torch.nn.functional.conv2d(x, self.weight)
+
+    model = FunctionalConv().train()
+    x = torch.randn(1, 2, 5, 5, requires_grad=True)
+
+    with pytest.raises(ValueError, match="aten.convolution_backward.default"):
```

Ruff (0.16.2)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)